tests/conftest.py

```diff
@@ -39,6 +63,25 @@ def docker(event_loop):
 
 
 @pytest.fixture
+def swarm(event_loop, docker):
```

Comment:
@barrachri I found this by looking on the web: https://stackoverflow.com/questions/12411431/pytest-how-to-skip-the-rest-of-tests-in-the-class-if-one-has-failed/12579625#12579625

I think that would solve partially our argument about test isolation. But I still think the enter/leave Swarm should be in a setUp/tearDown.